### ISSUE TITLE
After a print has started, nozzle fluctuates around target temp and takes forever to start

### ISSUE THREAD

**Issue by Daitaro302** [KevinOConnor/klipper#2773]
Hello all!

I'm noticing something that I would honestly like to see act differently. After I start a print, the printer heats up, like normal. However, once it reaches the temp, it seriously just hovers around it, both above and below. If I'm targeting 220 for instance, it will hit 220, go up to 225, down to 218, back up to 227, and literally just float around the target for up to two minutes or so before starting the print. This isn't new; it has done this since I changed over to klipper. I'm running a cr-10s, with modifications to the printer itself - none related here. This printer prints *beautifully* , this problem *only* happens during warm-up. I don't have an issue with it really, but it's very inconvenient having to sit there and clean the nozzle each print. Seriously, it hovers around for so long that by the time the print starts, there's a dime-sized blob hanging around the nozzle.

Are there some better targeting options I could try out in the config files? I don't have any fans blowing around the printer, the humidity is low; this isn't a crazy environment here. Very normal. 

Thanks, guys!

**Comment by klipper-gitissuebot** [comment]
Hi @Daitaro302,

It did not look like there was a Klipper log file attached to this ticket. The log file has been engineered to answer common questions the Klipper developers have about the software and its environment (software version, hardware type, configuration, event timing, and hundreds of other questions).

Unfortunately, too many people have opened tickets without providing the log. That consumes developer time; time that would be better spent enhancing the software. If this ticket references an event that has occurred while running the software then the Klipper log must be attached to this ticket. Otherwise, this ticket will be automatically closed in a few days.

For information on obtaining the Klipper log file see: https://github.com/KevinOConnor/klipper/blob/master/docs/Contact.md

The log can still be attached to this ticket - just add a comment and attach the log to that comment.

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being.

**Comment by Hywelmartin** [comment]
if a PID tune doesn't help.. try lower the _max_power: 1.0_ to the hotend and/or _pid_integral_max:_
lowering _smooth_time: 2.0_ might help

**Comment by pusewicz** [comment]
@Daitaro302 Please make sure you calibrate the PID:

https://github.com/KevinOConnor/klipper/blob/master/docs/Config_checks.md#calibrate-pid-settings

**Comment by klipper-gitissuebot** [comment]
It appears the Klipper log file was not provided.
I'm closing this now, sorry. Feel free to attach the log to this ticket sometime in the future, or create a new issue once you can provide the log file.
